environments.go

```diff
@@ -208,17 +208,19 @@ func (s *EnvironmentsService) DeleteEnvironment(pid interface{}, environment int
 //
 // GitLab API docs:
 // https://docs.gitlab.com/ee/api/environments.html#stop-an-environment
-func (s *EnvironmentsService) StopEnvironment(pid interface{}, environmentID int, options ...RequestOptionFunc) (*Response, error) {
+func (s *EnvironmentsService) StopEnvironment(pid interface{}, environmentID int, options ...RequestOptionFunc) (*Environment, *Response, error) {
 	project, err := parseID(pid)
 	if err != nil {
-		return nil, err
+		return nil, nil, err
 	}
 	u := fmt.Sprintf("projects/%s/environments/%d/stop", PathEscape(project), environmentID)
 
 	req, err := s.client.NewRequest(http.MethodPost, u, nil, options)
 	if err != nil {
-		return nil, err
+		return nil, nil, err
 	}
 
-	return s.client.Do(req, nil)
+	env := new(Environment)
+	resp, err := s.client.Do(req, env)
+	return env, resp, err
```

Comment:
```suggestion
	env := new(Environment)
	resp, err := s.client.Do(req, env)
	if err != nil {
		return nil, resp, err
	}
	
	return env, resp, nil
```
For consistency... Also noticed this in this file the returns always return `err` even when the `err` is already checked. So feel free (please do 😉 ) to update the other return statements as well 😊

Reply:
Hmm not only this file I see. 2 other random files I checked also always return err, even when it's already checked. 
Want me to do a search replace in a new PR maybe?

Reply:
Sure... I prefer to keep things consistent, so please do 😊

Reply:
https://github.com/xanzy/go-gitlab/pull/1710